tfx/examples/custom_components/slack/setup.py

```diff
@@ -24,8 +24,8 @@ def _make_required_install_packages():
   # six, and protobuf) with TF.
   return [
       'slackclient>=2.0.0,<2.0.1',
-      'tfx>=0.14.0,<0.15.0',
```

Comment:
I'm fine if we change this to tfx==0.14.0dev since this will now only work with head.